kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -348,18 +341,22 @@
     width: ($iphone-5-width - 20)px
 
   .terms
```

Comment:
Still need to move this .terms style to the develop branch.

Reply:
It's already in there - If anything, I can just revert the changes or remove the styles altogether?

Reply:
Yes remove it from this branch.